Ports/ssmtp/patches/0004-Use-generic-default-configuration.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fabian Dellwing <fabian.dellwing@gmail.com>
+Date: Fri, 28 Apr 2023 07:43:47 +0200
+Subject: [PATCH] Use generic default configuration
+
+Make the installation fully non-interactive and create a sane
+default ssmtp.conf
+---
+ generate_config | 37 ++-----------------------------------
+ 1 file changed, 2 insertions(+), 35 deletions(-)
+
+diff --git a/generate_config b/generate_config
+index a16efd46b88829a987935be2f1a920416aa19599..d09cd177d3a4c057d93e96d225a12fee4a2b9026 100755
+--- a/generate_config
++++ b/generate_config
+@@ -1,37 +1,4 @@
+-#!/bin/sh -e
+-
+-#
+-# Figure out the system's mailname
+-#
+-
+-syshostname=`hostname --fqdn`
+-if test -f /etc/mailname
+-then
+-	mailname="`head -1 /etc/mailname`"
+-fi
+-
+-if test -z "$mailname"
+-then
+-	mailname=$syshostname
+-fi
+-
+-echo "Please enter the mail name of your system."
+-echo "This is the hostname portion of the address to be shown"
+-echo "on outgoing news and mail messages headers."
+-echo "The default is $syshostname, your system's host name."
+-echo
+-echo -n "Mail name [$syshostname]: "
+-read mailname
+-echo
+-
+-echo -n "Please enter the SMTP port number [25]: "
+-read smtpport
+-if test -z "$smtpport"
+-then
+-	mailhub=$mailhub
+-else
+-	mailhub="$mailhub:$smtpport"
+-fi
++#!/bin/bash -e
+ 
+ #
+ # Generate configuration file
+@@ -56,7 +23,7 @@ mailhub=mail
+ # Where will the mail seem to come from?
+ #rewriteDomain=`echo -n $mailname`
+ # The full hostname
+-hostname=`hostname --fqdn`
++hostname=courage
+ EOF
+ 
+ echo
```

Comment:
If I replace the `sed -i`s so the patching can continue, this patch fails for me with:
```
[ssmtp/patch_internal] patching file generate_config
[ssmtp/patch_internal] 1 out of 2 hunks failed--saving rejects to generate_config.rej
Error in step ssmtp/patch_internal (status=1)
```
I don't think it should be caused by either replace action, so that's why I mention it here anyway.

Reply:
It is caused by applying `02-557725-solaris.patch`.